### PR TITLE
[HW 7.2]: Added Rubocop

### PR DESCRIPTION
**Pull request by joergenn**
## What was done in this PR?
  - Added the rubocop gem.
  - Did `bundle install`.

### CODE REVIEW

File: .idea/.generators [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (NickUkraine24) [inline .idea/.generators:1]: Please, add `.idea` to `.gitignore` because we don't need to commit `.idea` files.

**Comment by joergenn** [comment]
Thanks for your review, successfully added .idea to .gitignore.

вт, 1 июн. 2021 г. в 18:20, Mykola Ilashchuk ***@***.***>:

**Comment by NickUkraine24** [comment]
Ok, but at this PR I don't see a resolve about the issue above.
Please resolve.

**Comment by joergenn** [comment]
Is the problem gone now? Do I have to click the merge pull request button after you've reviewed PR? By the way, thanks for the review

**Pull request merged**